### PR TITLE
Add more detailed descriptions of endpoints/ports

### PR DESCRIPTION
**Pull request by mjdesa**

### CODE REVIEW

**Comment by mjdesa** [comment]
@beckettsean @gunnaraasen @rkuchan feedback please :)

File: content/docs/v0.9/concepts/api.md [hunk 1]
```diff
@@ -1,16 +1,47 @@
...
-| /write                           | POST    |
-| /ping                            | GET     |
-| /ping                            | HEAD    |
-| /data/process_continuous_queries | POST    |
 
+### /ping
+
+The ping endpoint accepts both `GET` and `HEAD` HTTP requests. There are no query parameters that can be passed to the endpoint. The response body is empty. The version of the InfluxDB server you issued the request to can be extracted through the `X-Influxdb-Version` field of the header.
+
+For example:
```
Comment (beckettsean) [inline content/docs/v0.9/concepts/api.md]: "For example, this is the `/ping` response from a server running version `0.9.4-nightly-548b898` of InfluxDB."

File: content/docs/v0.9/concepts/api.md [hunk 2]
```diff
@@ -1,16 +1,47 @@
...
+For more information on the `/query` endpoint see the [Querying Data](/docs/v0.9/guides/querying_data.html) section of our docs.
+
+### /write
+For more information on the `/write` endpoint see the [Writing Data](/docs/v0.9/guides/writing_data.html) section of our docs.
+
+## Ports
+
+### HTTP API PORT
+
+By default the InfluxDB HTTP API listens on port `8086`.
```
Comment (beckettsean) [inline content/docs/v0.9/concepts/api.md]: The `/ping`, `/write`, and `/query` endpoints are all part of the HTTP API.

**Comment by beckettsean** [comment]
a few notes, otherwise :+1:

**Comment by mjdesa** [comment]
fixes https://github.com/influxdb/influxdb.com/issues/236

**Comment by mjdesa** [comment]
fixes https://github.com/influxdb/influxdb.com/issues/175

**Pull request merged**